### PR TITLE
adds support for post in apiargs descriptions

### PR DESCRIPTION
**Pull request by schmalliso**

### CODE REVIEW

**Comment by schmalliso** [comment]
@tychoish Updated!

File: giza/giza/content/apiargs/models.py [hunk 1]
```diff
@@ -63,16 +64,15 @@ def arg_name(self, value):
...
-            if self.optional is True:
-                return 'Optional. ' + self.state['description']
-            else:
-                return self.state['description']
+        output = []
+
+        for element in ("pre", "description", "post"):
+            if element in self.state:
+                output.append(self.state[element])
+                output.append("\n\n")
```
Comment (tychoish) [inline giza/giza/content/apiargs/models.py]: one new line here and one in the join. 

add a blank line here.

File: giza/giza/content/apiargs/models.py [hunk 2]
```diff
@@ -63,16 +64,15 @@ def arg_name(self, value):
...
+        if self.optional is True:
+            output[0] = "Optional. " + output[0]
```
Comment (tychoish) [inline giza/giza/content/apiargs/models.py]: insert blank line.

**Comment by tychoish** [comment]
lgtm modulo nits

**Comment by schmalliso** [comment]
Done and done! Merged.

**Pull request merged**